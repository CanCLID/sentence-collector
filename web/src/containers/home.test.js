import React from 'react';
import { render } from '@testing-library/react';

import Home from './home';

test('should render home', () => {
  const { container } = render(<Home/>);
  expect(container).toMatchSnapshot();
});
