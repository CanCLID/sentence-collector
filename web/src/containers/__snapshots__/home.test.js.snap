// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home 1`] = `
<div>
  <div>
    <h1>
      Welcome to the Common Voice Sentence Collector
    </h1>
    <p>
      This is a website where we collect and review sentences for 
      <a
        href="https://commonvoice.mozilla.org/"
      >
        Common Voice
      </a>
      . Check the Common Voice 
      <a
        href="https://common-voice.github.io/community-playbook/"
      >
        Playbook
      </a>
       to find out more about the Common Voice project.
    </p>
  </div>
</div>
`;
